README.md

```diff
@@ -78,13 +78,15 @@ This will set up a new application on your Heroku account using this repo as a t
 
 -->
 
-### Import to Replit
+### Import to CodeSandbox
 
-Create a Replit account at [Replit - Sign up](https://replit.com/signup), and then click on this button:
+Log in or register on [CodeSandbox - Sign in](https://codesandbox.io/signin).
 
-[![Run on Replit](https://replit.com/badge?caption=Import%20to%20Replit)](https://replit.com/github/upleveled/express-guest-list-api-memory-data-store)
+Change the `express-guest-list-api-memory-data-store` repo URL to include "box" (`githubbox.com` instead of `github.com`) and hit return. This will open the project on CodeSandbox.
 
-This will import the code from this repo to a new repl on Replit. Once it has been imported, run the application using the green **Run** button at the top center of the screen.
+Click on "Fork" in the top right corner of the page to create your own personal copy of the project.
+
+Beside "Fork", click "Share", open the "Visibility" dropdown, select "Public" and click "Move Devbox" to allow access to your devbox from anyone.
```

Comment:
hm, with 4 paragraphs, it's looking like this could be a 4-item numbered list

Reply:
Updated to a numbered list
[638883a](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294/commits/638883aee8d09decb48efaf1a5e23773d8ea4fb4)

Reply:
I think Markdown numbered lists don't have empty lines between them often (not sure if this causes formatting weirdness) - let's remove those

Reply:
Removed the line breaks between the numbered list
[12defd6](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294/commits/12defd697883796b0b81f27eafdc0264221034b0)